polyfill/test/ecmascript.mjs

```diff
@@ -64,6 +65,61 @@ describe('ECMAScript', () => {
       });
     }
   });
+
+  describe('GetAvailableNamedTimeZoneIdentifier', () => {
+    it('Case-normalizes time zone IDs', () => {
+      // eslint-disable-next-line max-len
+      // curl -s https://raw.githubusercontent.com/unicode-org/cldr-json/main/cldr-json/cldr-bcp47/bcp47/timezone.json > cldr-timezone.json
+      const cldrTimeZonePath = new URL('./cldr-timezone.json', import.meta.url);
```

Comment:
Do we really need to be this exhaustive in this test? Anba already added test262 tests that are pretty exhaustive. I'd prefer not to carry the whole copy of the CLDR data if we can avoid it.

Reply:
The purpose of this test is to validate that the case-normalization of IDs in the polyfill works as expected for every available ID. Case-normalization is not (before proposal-canonical-tz) observable behavior, so we can't rely on Test262 to validate it. Maybe remove this later if that proposal is approved?

Reply:
Sure, that's fine, although if it were just up to me I just wouldn't have an exhaustive test until the proposal-canonical-tz is ready to be part of test262. It seems like a lot to have to keep up to date indefinitely, for relatively little gain. OK to move it into test262 soon after proposal-canonical-tz reaches stage 3?

Reply:
Yep, that sounds like the right plan. 🙏